index.html

```diff
@@ -53,5 +53,226 @@ <h3 class="job-title">
             <div id="background-img" class="background-img"></div>
         </div>
     </section>
+    <!-- Works section -->
+    <section>
+        <div class="works">
+            <div class="title">
+                <h2 class="recentWorks">
+                    My Recent Works
+                </h2>
+                <img src="images/line.png" alt="line">
+            </div>
+            <div class="work1">
+                <img src="images/Img Placeholder.png" alt="image">
+                <h3 class="work1Title">
+                    Multi-Post Stories
+                </h3>
+                <p class="work1Description">
+                    A daily selection of privately personalized reads; no accounts or 
+                    sign-ups required. has been the industry's standard dummy text ever 
+                    since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="work1Languages">
+                    <li> <button class="work1Button" type="button">CSS</button> </li>
+                    <li> <button class="work1Button" type="button">HTML </button> </li>
+                    <li> <button class="work1Button" type="button">Bootstrap</button> </li>
+                    <li> <button class="work1Button" type="button">Ruby</button> </li>
+                </ul>
+                <button class="work1Link">See project</button>
```

Comment:
- [ ] kindly consider adding  button styling like: Active, Hover and Pressed on all your buttons on the page  this will improve the user experience you may check on the below image 

![dd](https://user-images.githubusercontent.com/60146030/227634129-fb518443-2f30-469a-bb58-95641389f0c0.PNG)